src/timer/web.rs

```diff
@@ -31,12 +34,15 @@ impl Drop for TimerHandle {
 fn timer_create(timeout: time::Duration, state: *const TimerState) -> TimerHandle {
     let timeout = timeout.as_millis() as u32;
 
-    let cb = wasm_bindgen::closure::Closure::once(move || unsafe {
+    let closure = wasm_bindgen::closure::Closure::once(move || unsafe {
         (*state).wake();
     });
-    let handle = setTimeout(&cb, timeout);
+    let timeout_id = setTimeout(&closure, timeout);
```

Comment:
Renamed because the `Closure` documentation calls _it_ a "handle" too; `timeoutID` is what MDN calls this: https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout

Reply:
Sounds good